pages/linux/kscreen-console.md

```diff
@@ -0,0 +1,32 @@
+# kscreen-console
+
+> KScreen Console is a CLI tool to query KScreen's status.
+> More information: <https://manned.org/kscreen-console>.
+
+- Show all outputs and configuration files to attach to a bug report:
+
+`kscreen-console bug`
+
+- Show paths to KScreen configuration files:
+
+`kscreen-console config`
+
+- Show display output information:
```

Comment:
Can we find a better description? The output looks famility to the `json` example.

Reply:
Hmm, I can't think of something immediately, does anyone else have an opinion?

Maybe something like?

```suggestion
- Show KScreen output information and configuration:
```

Example output:
```
START: Requesting Config
kscreen.kwayland: Loading Wayland backend.
Received config. Took 64 milliseconds
Screen:
	maxSize: QSize(64000, 64000)
	minSize: QSize(0, 0)
	currentSize: QSize(360, 720)

-----------------------------------------------------

Id:  1
Name:  "unknown DSI-1-unknown"
Type:  "Panel (Laptop)"
Connected:  true
Enabled:  true
Primary:  true
Rotation:  KScreen::Output::None
Pos:  QPoint(0,0)
MMSize:  QSize(68, 136)
FollowPreferredMode:  false
Size:  QSize(720, 1440)
Scale:  2
Clones:  None
Mode:  "0"
Preferred Mode:  "0"
Preferred modes:  ("0")
Modes: 
	 "0"    "720x1440@60"   QSize(720, 1440)   60.006
EDID Info: 
	Unavailable
```

---

Also I'm sorry, I didn't understand "The output looks famility to the json example".